google-cloud-storage/src/test/resources/META-INF/native-image/com/google/cloud/storage/native-image.properties

```diff
@@ -0,0 +1,26 @@
+Args = \
```

Comment:
The file that tells build-time initialization of classes of the library shouldn't be in test. Because the file is not shipped as part of the library.

As discussed today, this file being under the test directory means the users would have to write something similar to use google-cloud-storage with native-image.

Reply:
Hm, the only reason why these configurations are initialized at build-time is because  `com.google.cloud.storage.conformance.retry.ITRetryConformanceTest` which references them is initialized at build-time (following[ this logic](https://github.com/googleapis/java-storage/pull/1305#discussion_r834555617)).

Reply:
That's my misunderstanding then. Sorry I missed the discussion. Would you add that comment in the file about the configuration being for the test classes?

Reply:
Now the `--initialize-at-build-time` doesn't have google-cloud-storage's main package. Nice.